pkg/core/xds/matched_policies.go

```diff
@@ -27,7 +27,9 @@ type MatchedPolicies struct {
 	TrafficRoutes      RouteMap
 
 	// Dataplane -> Policy
-	TrafficTrace *core_mesh.TrafficTraceResource
+	// Actual Envoy Configuration is generated without taking these policies into account
```

Comment:
This comment should be applied only to the `ProxyTemplate` field (not to `TrafficTrace`). Also, you can add a similar comment for `TrafficRoute` field

Reply:
Do you mean the same comment to be duplicated for `TrafficTrace`?

Reply:
No, TrafficTraces are fine, aren't they? I'm talking about TrafficRoutes, you added this field [here](https://github.com/kumahq/kuma/pull/3735) and this field is used only for Inspect API, it's not taken into account by proxy generator

Reply:
understood